packages/widget/src/components/Chains/VirtualizedChainList.tsx

```diff
@@ -38,13 +41,17 @@ export const VirtualizedChainList = ({
   const { setFieldValue } = useFieldActions()
   const selectedChainIdRef = useRef(selectedChainId) // Store the initial selected chain ID to scroll to it once chains are loaded
   const hasScrolledRef = useRef(false)
-  const [pinnedChains, setPinnedChain, isAllNetworks, setIsAllNetworks] =
-    useChainOrderStore((state) => [
+  const [pinnedChains, setPinnedChain, setIsAllNetworks] = useChainOrderStore(
+    (state) => [
       state.pinnedChains,
       state.setPinnedChain,
-      state.isAllNetworks,
       state.setIsAllNetworks,
-    ])
+    ]
+  )
+  const isAllNetworks = useChainOrderStore(
+    (state) => state[`${formType}IsAllNetworks`]
+  )
```

Comment:
We should put this inside the array above instead of calling store one more time 🙂